From 6ee661f3845cbd474a22a934267b2bae2bbb3add Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Thu, 10 Aug 2023 15:35:57 +0200
Subject: [PATCH 056/120] spi: stm32_ospi: Remove unused stm32_ospi_ids struct

Remove unused stm32_ospi_ids struct.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/u-boot.git d03564716d86ff0be4576edf31e9297c1d4872e1 spi: stm32_ospi: Remove unused stm32_ospi_ids struct]
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Change-Id: Ib6f3a8e17adb157614eb75a69e95173117e50ccc
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/u-boot/+/322441
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/spi/stm32_ospi.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/spi/stm32_ospi.c b/drivers/spi/stm32_ospi.c
index 8ee1b3d1e70..18ac48100d5 100644
--- a/drivers/spi/stm32_ospi.c
+++ b/drivers/spi/stm32_ospi.c
@@ -469,15 +469,9 @@ static const struct dm_spi_ops stm32_ospi_ops = {
 	.mem_ops	= &stm32_ospi_mem_ops,
 };
 
-static const struct udevice_id stm32_ospi_ids[] = {
-	{ .compatible = "st,stm32mp25-ospi" },
-	{ }
-};
-
 U_BOOT_DRIVER(stm32_ospi) = {
 	.name = "stm32_ospi",
 	.id = UCLASS_SPI,
-	.of_match = stm32_ospi_ids,
 	.ops = &stm32_ospi_ops,
 	.priv_auto = sizeof(struct stm32_ospi_priv),
 	.probe = stm32_ospi_probe,
-- 
2.47.2

